kinta-lib/src/main/kotlin/com/dailymotion/kinta/integration/git/model/Model.kt

```diff
@@ -0,0 +1,17 @@
+package com.dailymotion.kinta.integration.git.model
```

Comment:
This should be in the `com.dailymotion.kinta.integration.gittool.model` package. Git itself doesn't really know about pull requests